CMakeLists.txt

```diff
@@ -519,10 +519,27 @@ add_subdirectory( "cmake-proxies" )
 # Conan uses find_package and does not set GLOBAL flag
 resolve_conan_dependencies()
 
+# This libraries have 3d party dependencies.
+# Moreover, portmixer is only available for as a part of
```

Comment:
portmixer is  only available "for as" ???

Reply:
Whoopsie